.github/workflows/landing-builds.yml

```diff
@@ -34,7 +34,7 @@ jobs:
       fail-fast: false
       max-parallel: 1
       matrix:
-        distro: [ubuntu, rocky, fedora, leap.15]
+        distro: [ubuntu, rocky, fedora, leap.15, alma.9]
```

Comment:
This is the matrix for building the docker caches which will then be used for a different step in this file, and for PR builds, so it's the wrong place to add this.

It should go with the other Alma build on L255.